scripts/config/org/icepear/echarts/origin/chart/treemap/treemap-series-option.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "TreemapSeriesOption",
+    "type": "interface",
+    "extends": [
+        "SeriesOption",
+        "TreemapStateOption",
+        "BoxLayoutOptionMixin",
+        "RoamOptionMixin",
+        "TreemapSeriesVisualOption"
+    ],
+    "fields": [
+        { "name": "type", "types": ["String"], "default": "treemap"},
+        { "name": "size", "types": ["Number[]", "String[]"]},
+        { "name": "sort", "types": ["Boolean"], "default": true},
+        { "name": "clipWindow", "types": ["String"]},
+        { "name": "squareRatio", "types": ["Number"]},
+        { "name": "leafDepth", "types": ["Number"]},
+        { "name": "drillDownIcon", "types": ["String"]},
+        { "name": "zoomToNodeRatio", "types": ["Number"]},
+        { "name": "nodeClick", "types": ["String"]},
+        { "name": "breadcrumb", "types": ["TreemapSeriesBreadCrumbOption"]},
```

Comment:
Let's use `TreemapSeriesBreadcrumbOption` with a lowercase `c` to keep consistent with `breadcrumb`.

Reply:
Looks like `TreemapSeriesBreadCrumbOption` only exists in chart/treemap, we can rename it to `BreadCrumbOption`.